swarm/storage/mru/metadata.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
the metadata chunk should be an ordinary swarm chunk. No need for any validation other than content address. Also no need for initial byte differentiator etc. Simplify the code.